lib/bridge.js

```diff
@@ -964,29 +969,33 @@ function queueAlgorithm(event) {
 }
 
 class EventQueue {
```

Comment:
For each of these classes it would be good to have a jsdoc explaining what they do.